napi-inl.h

```diff
@@ -3618,6 +3618,275 @@ inline void AsyncWorker::OnWorkComplete(
   delete self;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// ThreadSafeFunction class
+////////////////////////////////////////////////////////////////////////////////
+
+// static
+template <typename ResourceString>
+inline ThreadSafeFunction ThreadSafeFunction::New(napi_env env,
+                                                  const Function& callback,
+                                                  ResourceString resourceName,
+                                                  size_t maxQueueSize,
+                                                  size_t initialThreadCount) {
+  return New(env, callback, Object(), resourceName, maxQueueSize,
+             initialThreadCount, nullptr);
```

Comment:
`nullptr` used here (and elsewhere) is ambiguous so shows up as `mismatched types ‘Context*’ and ‘std::nullptr_t’` when the intended overload is considered as a candidate

apparently need an explicitly overload with `std::nullptr_t` to resolve it